modules/client-configuration/pages/client-proxy-script.adoc

```diff
@@ -0,0 +1,71 @@
+[[script-client-proxy]]
+= Register Clients  to a Proxy with a Bootstrap Script
+
+
+You can register clients through the {productname} Proxy with a bootstrap script.
```

Comment:
Clients can be registered through the {productname} Proxy with a bootstrap script.

Reply:
I'd much rather see this phrase, than "You can..." at the very beginning of a paragraph. It puts an emphasis on the action we talk about, not on the user.